game/src/contrib/systems/CollisionSystem.java

```diff
@@ -19,18 +20,51 @@ public CollisionSystem() {
         super(CollideComponent.class);
     }
 
+    /**
+     * Test every CollideEntity with each other CollideEntity for collision.
+     *
+     * <p>The collision check will be performed only once for a given tuple of entities. When entity
+     * A does collide with entity B it also means B collides with A.
```

Comment:
```suggestion
     * <p>The collision check will be performed only once for a given tuple of entities. 
     * When entity A does collide with entity B it also means B collides with A.
```

Reply:
Hmmm, macht ja auch Sinn.

Ich bin halt nur gestolpert: Die erste Zeile hört irgendwie komisch auf, und die zweite scheint neu anzufangen (großes "A" als erstes "Wort"). Erst dachte ich, ich hätte beim Vorschlagen in GH irgendwas kaputt gemacht ...

@AHeinisch Dann vielleicht sowas in der Art von "<p>The collision check will be performed only once for a given tuple of entities, i.e. when entity A does collide with entity B it also means B collides with A."?

Reply:
Habe es jetzt mal übernommen und zwar "The collision check will be performed only once for a given tuple of entities, i.e. when entity A does collide with entity B it also means B collides with A." und es mit spotless formatiert